client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -240,8 +240,6 @@ public void start() throws MQClientException {
                     this.pullMessageService.start();
                     // Start rebalance service
                     this.rebalanceService.start();
-                    // Start push service
-                    this.defaultMQProducer.getDefaultMQProducerImpl().start(false);
```

Comment:
It will be used to send retry messages if exception happens.
https://github.com/apache/rocketmq/blob/df3f8de4c03a282de052b130ae368ab00faca039/client/src/main/java/org/apache/rocketmq/client/impl/consumer/DefaultMQPushConsumerImpl.java#L512

![image](https://user-images.githubusercontent.com/46882838/165924585-a98b5be4-c9a1-45f2-b7e7-9e5f1d0e41a6.png)